### DISCUSSION TITLE
"For you" feed suggestion: aggregate user engagement with your repos

### DISCUSSION THREAD

**Discussion by Tyrrrz** [community/community#13285] [Other Feature Feedback, Questions, & Ideas]
Currently, the only way the original feed is useful to me is that it shows when other people star my repos. I find this information useful as it shows spikes of interest for particular repositories as well as just public interest in general.

One way it could probably be added to the new feed is by aggregating that data. I don't really care who stars my repos, as much as I care about how many people do in a particular timespan.

For instance, one way it could be presented is:

```
[-----------------------------------------]
|    Repo name                            |
|    Received 69 stars in the last day    |
[-----------------------------------------]
```

**Comment by sntwandy** [comment]
It's a good option to consider.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
